src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -99,11 +100,16 @@ public class OpensearchSinkConnectorConfig extends AbstractConfig {
     private static final String KEY_IGNORE_DOC =
             "Whether to ignore the record key for the purpose of forming the Opensearch document ID."
                     + " When this is set to ``true``, document IDs will be generated as the record's "
-                    + "``topic+partition+offset``.\n Note that this is a global config that applies to all "
-                    + "topics, use ``" + TOPIC_KEY_IGNORE_CONFIG + "`` to override as ``true`` for specific "
+                    + "``topic+partition+offset`` or a strategy set in ``" + KEY_IGNORE_ID_STRATEGY_CONFIG + "``.\n"
+                    + "Note that this is a global config that applies to all topics, use "
```

Comment:
I think we need to change the documentation here, basically we are splitting the setting into two: the intent (ignore record key) and the generation (how to generate the key). 

```
"Whether to ignore the record key for the purpose of forming the Opensearch document ID."
 + " When this is set to ``true``, document IDs will be generated according to the ``" 
 + KEY_IGNORE_ID_STRATEGY_CONFIG + "`` strategy.\n" 
 + "Note that this is a global config that applies to all topics, use ``"
 + TOPIC_KEY_IGNORE_CONFIG + "`` to apply `" + KEY_IGNORE_ID_STRATEGY_CONFIG 
 + "` strategy for specific topics only.";
```

May be something along these lines. The default `KEY_IGNORE_ID_STRATEGY_CONFIG` should be ``topic+partition+offset``.